pkgs/development/web/cypress/default.nix

```diff
@@ -1,24 +1,24 @@
-{ stdenv, lib, fetchzip, autoPatchelfHook, xorg, gtk2, gnome2, gtk3, nss, alsaLib, udev, unzip, wrapGAppsHook }:
+{ stdenv, lib, fetchzip, autoPatchelfHook, xorg, gtk2, gnome2, gtk3, nss
+, alsaLib, udev, unzip, wrapGAppsHook }:
 
 stdenv.mkDerivation rec {
   pname = "cypress";
-  version = "6.0.0";
+  version = "6.3.0";
 
   src = fetchzip {
     url = "https://cdn.cypress.io/desktop/${version}/linux-x64/cypress.zip";
-    sha256 = "0hii7kp48ba07gsd521wwl288p808xr2wqgk1iidxkzj2v6g71by";
+    sha256 = "AvZtCJCcA5RoIiSeSmFWsNqQdiAkrzcGNaMj2lU5oKY=";
   };
 
+  passthru.updateScript = ./update.sh;
+
   # don't remove runtime deps
   dontPatchELF = true;
 
   nativeBuildInputs = [ autoPatchelfHook wrapGAppsHook ];
 
-  buildInputs = with xorg; [
-    libXScrnSaver libXdamage libXtst
-  ] ++ [
-    nss gtk2 alsaLib gnome2.GConf gtk3 unzip
-  ];
+  buildInputs = with xorg; [ libXScrnSaver libXdamage libXtst ]
+    ++ [ nss gtk2 alsaLib gnome2.GConf gtk3 unzip ];
```

Comment:
Is unzip run at build time? If so it needs to go to nativeBuildInputs.

Reply:
That's already changed in master

Reply:
I've rebased this PR, cutting out the changes to the version number & hash, leaving only the addition of the UpdateScript.